app/models/user/communication_preferences.rb

```diff
@@ -1,6 +1,18 @@
 class User::CommunicationPreferences < ApplicationRecord
   belongs_to :user
 
+  def self.keys
```

Comment:
@iHiD I'll leave this to you. Maybe there's a better way to do it dynamically.

Reply:
Nah, this is fine. Thanks!